### PR TITLE
Modified url for Critical Path Updates on home page

### LINKED ISSUE #2094: The "Latest Critical Path Updates in Need of Testing" link isn't for critpath updates
The home page has a link "Latest Critical Path Updates in Need of Testing", but if you click it it just takes you to all testing updates, not critical path updates.

This was reported by @AdamWill (thanks!)

### PR DESCRIPTION
**Pull request by ghost**
fixes #2094

Signed-off-by: Vismay Golwala <vgolwala@redhat.com>

### CODE REVIEW

**Comment by cverna** [comment]
jenkies test

**Review by bowlofeggs — APPROVED** [review]

File: bodhi/tests/server/views/test_generic.py [hunk 1]
```diff
@@ -53,6 +53,16 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    def test_critical_update_link_home(self):
+        update = Update.query.first()
+        update.critpath = True
+        update.status = UpdateStatus.testing
+        self.db.commit()
+
```
Comment (bowlofeggs) [inline bodhi/tests/server/views/test_generic.py]: ```flake8``` is failing due to whitespace on this line:

```
pip	./bodhi/tests/server/views/test_generic.py:62:1: W293 blank line contains whitespace
pip	        
pip	^
```

File: bodhi/tests/server/views/test_generic.py [hunk 2]
```diff
@@ -53,6 +53,16 @@ def test_home(self):
...
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
+    def test_critical_update_link_home(self):
+        update = Update.query.first()
+        update.critpath = True
+        update.status = UpdateStatus.testing
+        self.db.commit()
+        
+        res = self.app.get('/',headers={'Accept': 'text/html'})
```
Comment (bowlofeggs) [inline bodhi/tests/server/views/test_generic.py]: ```flake8``` fails due to a missing space after the ,:

```
pip	./bodhi/tests/server/views/test_generic.py:63:31: E231 missing whitespace after ','
pip	        res = self.app.get('/',headers={'Accept': 'text/html'})
pip	                              ^
```

File: bodhi/tests/server/views/test_generic.py [hunk 3]
```diff
@@ -53,6 +53,16 @@ def test_home(self):
...
+        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
```
Comment (bowlofeggs) [inline bodhi/tests/server/views/test_generic.py]: ```flake8``` wants a space after the , on this line and also wants the line to be less than 101 characters:

```
pip	./bodhi/tests/server/views/test_generic.py:64:42: E231 missing whitespace after ','
pip	        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
pip	                                         ^
pip	./bodhi/tests/server/views/test_generic.py:64:101: E501 line too long (122 > 100 characters)
pip	        self.assertRegexpMatches(str(res),'status=testing&critpath=True.*Latest Critical Path Updates in Need of Testing')
pip	          
```

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

File: bodhi/tests/server/views/test_generic.py [hunk 4]
```diff
@@ -53,6 +53,16 @@ def test_home(self):
         self.assertIn('Log out', res)
         self.assertIn('Fedora Update System', res)
 
+    @unittest.skipIf(six.PY3, 'Not working with Python 3 yet')
```
Comment (bowlofeggs) [inline bodhi/tests/server/views/test_generic.py]: This test should work with Python 3 I think - let's see if we can remove this decorator and get the test to pass.

File: bodhi/tests/server/views/test_generic.py [hunk 5]
```diff
@@ -53,6 +53,16 @@ def test_home(self):
...
+    def test_critical_update_link_home(self):
+        update = Update.query.first()
+        update.critpath = True
+        update.status = UpdateStatus.testing
+        self.db.commit()
+        res = self.app.get('/', headers={'Accept': 'text/html'})
```
Comment (bowlofeggs) [inline bodhi/tests/server/views/test_generic.py]: I recommend putting empty lines before and after this line. It's not a required style so you can keep it this way if you prefer, but I find it to be nice to break tests into three code blocks, most of the time:

0. Test set up.
1. Execute the code you are testing.
2. Assert that the right thing happened.

By adding spaces above and below this line, it makes it easier to spot the three segments of the test, and highlights that we are testing the call to ```/```.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

**Review by bowlofeggs — APPROVED** [review]
Looks good!

**Pull request merged**